<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GlassMaker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-GlassMaker-Desktop_Qt_6_2_0_MinGW_64_bit-Debug/Everaldo-Crystal-Clear-App-miscellaneous.ico</normaloff>../build-GlassMaker-Desktop_Qt_6_2_0_MinGW_64_bit-Debug/Everaldo-Crystal-Clear-App-miscellaneous.ico</iconset>
  </property>
  <widget class="QListWidget" name="listaSkladnikow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listaSurowcow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>20</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybrane skladniki</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybrane surowce</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioMasowe">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>% Masowe</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioMolowe">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>% Molowe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz rodzaj jak będziesz chciał przeliczać podane wartości</string>
   </property>
  </widget>
  <widget class="QPushButton" name="przyciskWyboruProcentow">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zatwierdz i wprowadz dane</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listaWartosci">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>131</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wprowadzone wartości</string>
   </property>
  </widget>
  <widget class="QPushButton" name="przyciskPoprawy">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Popraw dane</string>
   </property>
  </widget>
  <widget class="QPushButton" name="przyciskObliczenia">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Wykonaj Obliczenia</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
